tools/structfield/testdata/src/has-warnings/main.go

```diff
@@ -31,7 +33,9 @@ type URLFieldName struct {
 }
 
 type URLFieldType struct {
-	Page          string `url:"page,omitempty"`          // want `change the "Page" field type to "\*string" in the struct "URLFieldType" because its tag uses "omitempty"`
-	PerPage       int    `url:"per_page,omitempty"`      // want `change the "PerPage" field type to "\*int" in the struct "URLFieldType" because its tag uses "omitempty"`
-	Participating bool   `url:"participating,omitempty"` // want `change the "Participating" field type to "\*bool" in the struct "URLFieldType" because its tag uses "omitempty"`
+	Page          string `url:"page,omitempty"`                   // want `change the "Page" field type to "\*string" in the struct "URLFieldType" because its tag uses "omitempty"`
+	PerPage       int    `url:"per_page,omitempty"`               // want `change the "PerPage" field type to "\*int" in the struct "URLFieldType" because its tag uses "omitempty"`
+	Participating bool   `url:"participating,omitempty"`          // want `change the "Participating" field type to "\*bool" in the struct "URLFieldType" because its tag uses "omitempty"`
+	PerPageZeros  int    `url:"per_page_zeros,omitzero"`          // want `change the "PerPageZeros" field type to "\*int" in the struct "URLFieldType" because its tag uses "omitzero"`
+	PerPageBoth   int    `url:"per_page_both,omitempty,omitzero"` // want `change the "PerPageBoth" field type to "\*int" in the struct "URLFieldType" because its tag uses "omitempty and omitzero"`
```

Comment:
OK, now here is where I'm a bit fuzzy - do we want to require using pointers for primitives when using `omitzero`? Do we want to allow `omitzero` at all for primitives?

I'm guessing the answer is "it depends on the particular field and its usage", but honestly I don't have experience with `omitzero`.

Thoughts?

cc: @stevehipwell - @alexandear - @zyfy29